src/themes/defaultTheme/forms.js

```diff
@@ -21,7 +21,7 @@ const forms = {
     text: colors.steel,
   },
   sizes: {
-    minHeight: sizes.heights.h16,
+    minHeight: sizes.heights.h20,
```

Comment:
`forms.sizes.minHeight`

Reply:
Are you sure? 🎃

Reply:
No but it should :/

Reply:
Nope cause that where we define `forms.sizes.minHeight`
A value can't define itself.